ksqldb-functional-tests/src/test/resources/query-validation-tests/null.json

```diff
@@ -76,6 +76,42 @@
         "type": "io.confluent.ksql.util.KsqlStatementException",
         "message": "Comparison with NULL not supported: NULL <> NULL"
       }
+    },
+    {
+      "name": "coalesce",
+      "statements": [
+        "CREATE STREAM INPUT (COL0 INT KEY, COL1 STRING, COL2 ARRAY<INT>) WITH (kafka_topic='test_topic', value_format='JSON');",
+        "CREATE STREAM OUTPUT AS SELECT COALESCE(COL0, 10, 20) AS A, COALESCE(COL1, 'x') AS B, COALESCE(COL2, ARRAY[10, 20]) AS C FROM INPUT;"
```

Comment:
can we add a test for `COALESCE(some_column)` (e.g. no constants)

Reply:
done